docs: add info about path to solc binary (#241)
foundry-rs/book: src/faq.md

Forge will try to auto-detect what Solidity compiler works for your project.

To use a specific Solidity compiler, you can set [`solc`][config-solc] in your [config file][config],
or pass `--use solc:<version>` to a Forge command that supports it (e.g. [`forge build`][forge-build]
or [`forge test`][forge-test]). 

> Paths to a solc binary are also accepted. To use a specific local solc binary, set set path as `solc = "<path to solc>"` in your config file or pass `--use "<path to solc>"`. 

Reviewer: ```suggestion
Paths to a solc binary are also accepted. To use a specific local solc binary, you can set `solc = "<path to solc>"` in your config file, or pass `--use "<path to solc>"`. 
```
